src/api/SFA.DAS.Provider.Events.Api/Controllers/SubmissionsController.cs

```diff
@@ -59,5 +60,39 @@ public SubmissionsController(IMediator mediator, ILogger logger)
                 return InternalServerError();
             }
         }
+
+        [Route("api/v2/submissions", Name = "GetLatestLearnerEventByStandard")]
+        [HttpGet]
+        public async Task<IHttpActionResult> GetLatestLearnerEventByStandard(long uln, long sinceEventId = 0)
+        {
+            try
+            {
+                _logger.Debug($"Processing GetLatestLearnerEventByStandard, uln={uln}, sinceEventId={sinceEventId}");
+
+                var queryResponse = await _mediator.SendAsync(new GetLatestLearnerEventByStandardQueryRequest
+                    {
+                        SinceEventId = sinceEventId,
+                        Uln = uln
+                    })
+                    .ConfigureAwait(false);
+                
+                if (!queryResponse.IsValid)
+                {
+                    throw queryResponse.Exception;

```

Comment:
I know this is the existing pattern in the solution but it's awful (and loses the stack trace!).  If there is an exception it should be thrown from the query rather than added to the response message and then thrown.